src/standalone/mod.rs

```diff
@@ -116,6 +120,46 @@ impl StandaloneRead {
             .collect::<Result<Vec<_>>>()?;
         Ok(())
     }
+
+    /// Retrive the invoice from the standalone bindle.
+    ///
+    /// This loads the invoice from disk and then parse it into an invoice.
+    /// Errors can result either from reading from disk or from parsing the TOML
+    pub async fn get_invoice(&self) -> Result<crate::invoice::Invoice> {
+        let data = tokio::fs::read(&self.invoice_file)
+            .await
+            .map_err(ClientError::Io)?;
+        let inv = toml::from_slice(&data)?;
+        Ok(inv)
+    }
+
+    /// Get the path to a parcel in a standalone bindle
+    pub fn parcel_data_path(&self, parcel_id: &str) -> PathBuf {
```

Comment:
Should there be an `invoice_data_path` as well for those who need access to the file path?

Reply:
That is a good question. There is an `invoice_file` public attribute on the struct that is the full path. Is there a good reason to also have that as a method?

My reasoning behind `parcel_data_path()` is that (a) it exists on the file provider, and (b) to construct the path you have to understand the structure of a standalone bindle.

Reply:
I forgot that we had the path publicly available. That should be fine then